.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*.*.*'
+  pull_request:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Prepare
+        id: prep
+        run: |
+          DOCKER_IMAGE=quay.io/external_storage/nfs-subdir-external-provisioner
```

Comment:
Can we make this image name configurable via arg as well?

Reply:
It's a good idea! Will do.